.storybook/stories/tree/tree-node.stories.ts

```diff
@@ -56,27 +56,97 @@ const defaultParameters: Parameters = {
     isExpandable: { control: { disable: true }, table: { disable: true } },
     isSelectable: { control: { disable: true }, table: { disable: true } },
     onKeyDown: { control: { disable: true }, table: { disable: true } },
+    TREE_NODE_STATE: { control: { disable: true }, table: { disable: true } },
```

Comment:
Why do we need this?

Reply:
We add "TREE_NODE_STATE" to the "argTypes" in order to hide/disable the control in the table, else it will appear as below:

![Screenshot 2024-03-01 at 11 02 56 AM](https://github.com/vmware-clarity/ng-clarity/assets/1913702/8b468c74-b8fc-4f64-8a43-6244dd9f88f1)

Reply:
Don't we only need it in the args object?

Reply:
It is in the `args` object. This line is needed to hide it from view. I agree with @andyfeds.